src/visGeometry/index.ts

```diff
@@ -157,9 +171,9 @@ class VisGeometry {
     private agentPdbsToDraw: PDBModel[];
 
     public constructor(loggerLevel: ILogLevel) {
-        this.cam1 = cloneDeep(DEFAULT_CAMERA_SPEC);
-        this.cam2 = cloneDeep(DEFAULT_CAMERA_SPEC);
-        this.cam3 = cloneDeep(DEFAULT_CAMERA_SPEC);
+        this.cam1 = { ...cloneDeep(DEFAULT_CAMERA_SPEC), orthographic: false };
+        this.cam2 = { ...cloneDeep(DEFAULT_CAMERA_SPEC), orthographic: false };
+        this.cam3 = { ...cloneDeep(DEFAULT_CAMERA_SPEC), orthographic: false };
```

Comment:
what are these three camera setups for? I only see them used in the setupGui code

Reply:
They're for saveable/reloadable camera states in the test gui:
![image](https://github.com/simularium/simularium-viewer/assets/53030214/87629f34-8375-4882-a72c-aa615c04b84d)